README.md

```diff
@@ -1,2 +1,46 @@
-# web-ethics-commitment
-Site source for webethicscommitment.com
+# The Web Ethics Commitment Project
+
+![Current GitHub Pages Deployment Status](https://github.com/DrunkenUX/web-ethics-commitment/actions/workflows/build.yml/badge.svg)
+
+Ethical design and development of projects that power the web is more important now than ever. As the footprint grows, and new tools and techniques become available, so, too, do the opportunities for abuse. This site is designed to outline guiding behavior for web professionals, and provide references to supporting materials, so that you can help others commit to creating a web that is better for all of us. ~~This repo provides the source for webethicscommitment.com.~~
+
+## To Do Notes
+
+Code and non-code stuff that needs to happen before we officially launch.
+
+- [ ] Need to actually add the secitons in question to the index page. We won't actually accept endorsements until the principles are finalized. Getting documentation out of the way now though.
+- [ ] Actually create the ethics email address.
+- [ ] Finalize and purchase an actual domain name before calling it out
+
+## Contributing
+
+### Add Your Endorsement of Our Ethical Principles
+
+#### If you're not familiar with Git
+
+This process will be easier for designers and others not familiar with making code edits and commits, however may be slower for getting your endorsement added to the site.
+
+ * [Log an Issue](https://github.com/DrunkenUX/web-ethics-commitment/issues) in this repo that indicates that you want to sign
```

Comment:
The approach I typically see with this kind of thing is to have people submit a PR that modifies a common file (`signatories.md` or something adding their name to the list)

Reply:
Yeah, I think this is a better approach, but I need to set it up so that that works as a partial include in the page, which I think will happen as a standalone issue, at which point, this can be updated.

Reply:
Done.